Description: split() is deprecated in PHP 7, causing the code to fail
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/cacti/+bug/1662027
Bug-Debian: https://bugs.debian.org/
Source: https://github.com/Cacti/cacti/commit/6b5931130770cdfa5baa68c43a913dcabf1c3970

Index: cacti/poller.php
===================================================================
--- cacti.orig/poller.php
+++ cacti/poller.php
@@ -451,11 +451,9 @@ while ($poller_runs_completed < $poller_
 
 		/* sleep the appripriate amount of time */
 		if ($poller_runs_completed < $poller_runs) {
-			list($micro, $seconds) = split(' ', microtime());
-			$plugin_start = $seconds + $micro;
+			$plugin_start = microtime(true);
 			api_plugin_hook('poller_bottom');
-			list($micro, $seconds) = split(' ', microtime());
-			$plugin_end = $seconds + $micro;
+			$plugin_end = microtime(true);
 			if (($sleep_time - ($plugin_end - $plugin_start)) > 0) {
 				usleep(($sleep_time - ($plugin_end - $plugin_start)) * 1000000);
 			}
